src/frontend/org/voltdb/LegacyHashinator.java

```
@@ -146,4 +146,9 @@ public int pHashToPartition(VoltType type, Object obj) {
         }
         return set;
     }
+
+    @Override
+    public int getPartitionFromHashedToken(int hashedToken) {
+        return java.lang.Math.abs(hashedToken % catalogPartitionCount);
```

You can throw unsupported operation exception here, because there is no hashed token for the legacy hashinator. It should never be called.